packages/core/src/features/cohort/tests/mockData.ts

```diff
@@ -13,6 +13,7 @@ export const MOCK_COHORTS: Cohort[] = [
       genesCount: 24568,
       mutationCount: 22233445566,
       ssmCaseCount: 23450,
+      cnvOrSsmCaseCount: 28400,
```

Comment:
@craigrbarnes I made these numbers up. Let me know if I should do something else.